config/names/first_female_slime.txt

```diff
@@ -0,0 +1,73 @@
+Anablop
+Bloopella
+Blopbela
+Blopbelle
+Bloppina
+Bub
+Bubbles
+Bubblette
+Bubbli
+Bubblia
+Bubbly
+Fizzaria
+Fizzete
+Fizzian
+Fizzina
+Fizzney
+Fizzy
+Flubbelyn
+Flubbney
+Flubbzina
+Gelinda
+Gelletina
+Gellina
+Gooenna
+Gooescari
+Goolinda
+Isabloppa
+Jelibelle
+Jellai
+Jellei
+Jellenna
+Jellette
+Jelli
+Jellianne
+Jellica
+Jellina
+Jelly
+Jiggly
+Marmalade
+Murkella
+Murkina
+Oozella
+Oozerina
+Oozina
+Pipoppy
+Pippop
+Popiline
+Popilla
+Popparia
+Poppaya
+Poppella
+Poppet
+Poppi
+Poppla
+Poppurina
+Poppy
+Skishella
+Skishelly
+Skishly
+Skuishi
+Skweesha
+Skwishly
+Slush
+Slusharia
+Slushi
+Slushia
+Slushibell
+Slushie
+Slushy
+Squisha
+Squishie
+Squishlee
+Bloppella
```

Comment:
This file has an empty line at the end. While this is normally what we want in DM files, this ain't what we want in config files like this. Here it'll cause DM to see an empty line and parse it as such. So there is a chance that an empty name will be picked for each of these files